internal/tasks/schedulertask_test.go

```diff
@@ -1,81 +1,128 @@
+// +build unit
```

Comment:
Additionally, we run tests using the `-race` flag. https://golang.org/doc/articles/race_detector#Usage

Reply:
had tried running it with count 100-1000, will check this, possibly will make sense to all all tests